pylintrc

```diff
@@ -1,6 +1,7 @@
 [MASTER]
 ignore=tests,generated,vendored_sdks,privates
 ignore-patterns=test.*,azure_devops_build.*
+ignore-paths=.*/command_modules/.*/aaz
```

Comment:
- The `ignore` parameter represents Files or directories to be skipped. They should be base names, not paths.
- The `ignore-patterns` parameter represents Files or directories matching the regular expression patterns are skipped. The regex matches against base names, not paths.
- The `ignore-paths` parameter represents Files or directories matching the regular expressions patterns to the ignore-list.
So we should use `ignore-paths` here.